requirements/requirements.txt

```diff
@@ -83,6 +83,8 @@ defusedxml==0.6.0
     #   social-auth-core
 dictdiffer==0.8.1
     # via openshift
+distro==1.5.0
+    # via -r /awx_devel/requirements/requirements.in
```

Comment:
You can/should regenerate this with 3.8. Some things will fall out (and you can remove the license files for them)

Or that can be a followup change.

Reply:
Will do in a subsequent PR yep